public/js/reader.js

```diff
@@ -109,7 +111,38 @@ Reader.loadImages = function () {
             if (Reader.infiniteScroll) {
                 Reader.initInfiniteScrollView();
             } else {
-                $(document).on("click.imagemap_change_pages", "#Map area", Reader.handlePaginator);
+                $(window).on('wheel', function(e) {
+                    if(Reader.isFullscreen) {
+                        if (e.originalEvent.deltaY > 0) Reader.changePage(-1);
+                        else Reader.changePage(1);
+                        return false;
+                    }
+                });
+                $(document).on('click', function(event) {
+                    var img_y_offset = $('a#display img').offset()['top'] + $('a#display img').height();
+                    if (event.pageY < img_y_offset && !$('#overlay-shade').is(':visible') ){
+                        var whereside = event.pageX < $(window).width()/2 ? 'left' : 'right';
+                        if (!$(event.target).closest('a#display').length) {
+                            whereside = 'outer-'+whereside;
+                        }
+                        switch (whereside) {
+                        case "outer-left":
```

Comment:
Both this and `outer-right` breaks clicking on the arrow icons to move pages;  
I personally think those cases should be removed, since:  
- They're hard to figure out and will probably just confuse users who activate them by mistake 
- If users want to get to the first or last page they can just click the matching arrow shortcuts. (there's much less use for first/last page compared to -1/+1)
- It makes the above bit of code that much harder to read 😅